.travis.yml

```diff
@@ -14,7 +14,7 @@ install:
 - pip install coveralls
 script:
 - python manage.py collectstatic > /dev/null
-- py.test --cov --no-migrations
+- py.test --cov
 - python manage.py makemigrations --dry-run --check --noinput
```

Comment:
@rmader is that ok to do that? i'm a little weary of this but we don't have '--no-migrations' in other projects and it was where this was failing

Reply:
Sound right to me - that option should only make things a bit faster under certain circumstances. And apparently it was renamed to `--nomigrations` :shrug: